plugin/al/lib/AL_USDMaya/AL/usdmaya/nodes/ProxyShape.cpp

```diff
@@ -1479,6 +1479,13 @@ void ProxyShape::loadStage()
           stageId = StageCache::Get().Insert(m_stage);
           outputInt32Value(dataBlock, m_stageCacheId, stageId.ToLongInt());
 
+          // Set the stage in datablock so it's ready in case it needs to be accessed
+          MObject data;
+          MayaUsdStageData* usdStageData = createData<MayaUsdStageData>(MayaUsdStageData::mayaTypeId, data);
+          usdStageData->stage = m_stage;
+          usdStageData->primPath = m_path;
+          MStatus status = outputDataValue(dataBlock, outStageData(), usdStageData);
```

Comment:
/maya-usd/plugin/al/lib/AL_USDMaya/AL/usdmaya/nodes/ProxyShape.cpp:1465:19: error: unused variable 'status' [-Werror,-Wunused-variable]
          MStatus status = outputDataValue(dataBlock, outStageData(), usdStageData);

Reply:
that should be fixed now. thanks!